src/Modules/Admin/Commands.Admin/GetPowerBIWorkspaceEncryptionStatus.cs

```diff
@@ -0,0 +1,76 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.PowerBI.Common.Api;
+using Microsoft.PowerBI.Common.Api.Workspaces;
+using Microsoft.PowerBI.Common.Client;
+
+namespace Microsoft.PowerBI.Commands.Admin
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    public class GetPowerBIWorkspaceEncryptionStatus : PowerBIClientCmdlet
+    {
+        public const string CmdletName = "PowerBIWorkspaceEncryptionStatus";
+        public const string CmdletVerb = VerbsCommon.Get;
+
+        public GetPowerBIWorkspaceEncryptionStatus() : base() { }
+
+        public GetPowerBIWorkspaceEncryptionStatus(IPowerBIClientCmdletInitFactory init) : base(init) { }
+
+        #region Parameters
+
+        [Parameter(Mandatory = true)]
+        public string Name { get; set; }
+
```

Comment:
Should we support ID too?

Reply:
May want to support a pipeline too where you can pass in a workspace

Reply:
It is a good suggestion but is not part of V1 private preview. I'll check with pm if he needs this before going public